dbms/src/Interpreters/Context.cpp

```diff
@@ -1489,6 +1491,14 @@ BackgroundProcessingPool & Context::getBackgroundPool()
     return *shared->background_pool;
 }
 
+BackgroundProcessingPool & Context::getMovePool()
+{
+    auto lock = getLock();
+    if (!shared->move_pool)
+        shared->move_pool.emplace(settings.move_pool_size);
```

Comment:
maybe names of threads & log channel inside BackgroundProcessingPool should be parametrized to distinguish moving & normal background pools?

Reply:
+